content/en/training/_index.html

```diff
@@ -79,36 +79,48 @@ <h2>Learn with the Linux Foundation</h2>
 
 <section id="get-certified">
   <div class="main-section padded">
-    <h2>Get Kubernetes Certified</h2>
-    <div class="col-container">
-      <div class="col-nav">
+    <center>
+      <h2>Get Kubernetes Certified</h2>
+    </center>
```

Comment:
Try this:
```suggestion
    <h2><center>Get Kubernetes Certified</center></h2>
```

_For another PR:_
Really we should update this page to use more Markdown and less HTML, and certainly no `<center>` elements!

Reply:
Yeah for this PR I will modify this to the above!
@sftim should I create another PR to update the other `center` tags from HTML to Markdown?

Reply:
@AnirudhPanda would you be willing to open an issue about converting this page to Markdown (there might be one already, mind - if so, that's OK, we can use the existing issue).